From 38dba3c42b83c45b7a596bbb96b4b58abcec3c44 Mon Sep 17 00:00:00 2001
From: Dongsheng Yang <dongsheng.yang@linux.dev>
Date: Sat, 16 Aug 2025 07:50:59 +0000
Subject: [PATCH] dm-pcache: add build flags for testing

Signed-off-by: Dongsheng Yang <dongsheng.yang@linux.dev>
---
 drivers/md/dm-pcache/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/md/dm-pcache/Makefile b/drivers/md/dm-pcache/Makefile
index 86776e4acad2..ecf62b56b7dd 100644
--- a/drivers/md/dm-pcache/Makefile
+++ b/drivers/md/dm-pcache/Makefile
@@ -1,3 +1,5 @@
 dm-pcache-y := dm_pcache.o cache_dev.o segment.o backing_dev.o cache.o cache_gc.o cache_writeback.o cache_segment.o cache_key.o cache_req.o
 
 obj-m += dm-pcache.o
+
+ccflags-y += -Werror -Wmaybe-uninitialized -Wuninitialized -Wformat=2
-- 
2.43.0

